content/python/concepts/sets/terms/isdisjoint/isdisjoint.md

```diff
@@ -0,0 +1,66 @@
+---
+Title: '.isdisjoint()'
+Description: 'Checks whether two sets contains a common Element in it.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Collections'
+  - 'Sets'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.isdisjoint()`** method checks whether two [sets](https://www.codecademy.com/resources/docs/python/sets) contains a common Element.If no common Element Exists then **`.isdisjoint()`** method returns `True` otherwise `False`.
```

Comment:
```suggestion
The **`.isdisjoint()`** method checks whether two [sets](https://www.codecademy.com/resources/docs/python/sets) contain a common element in them. If no common element exists, then the method returns `True`. Otherwise, it returns `False`.
```